src/helpers/noti.js

```diff
@@ -0,0 +1,7 @@
+const io = require('socket.io').listen(5000);
+
+io.sockets.on('connection', function (socket) {
+  socket.on('connection name', function (user) {
+    io.sockets.emit('new user', `${user.name} has joined.`);
+  });
+});
```

Comment:
This is also not need, even not tested its dropping coverage. If a message is needed it could be in that other file.

Reply:
Sure, worked on.